spreadsheets/class/buildings.csv

```diff
@@ -1 +1,16 @@
-kind,class
+location,area::int,height::float,volume::int,landuse_kind,class
+!underground,>=100000,*,*,*,1
```

Comment:
Remove underground from this CSV and move that zoom 14+ only logic to the query filter?